index.js

```diff
@@ -5,6 +5,306 @@ const portfolio = document.querySelector('#portfolio-mobile');
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+  },
+  {
+    id: 4,
+    url: 'assets/img/Snapshoot-Portfolio.png',
+    name: 'Keeping track of hundreds of components ',
+    linkVersion: '#',
+    linkSource: '#',
+    tecnologies: {
+      tech1: 'codekit',
+      tech2: 'github',
+      tech3: 'JavScript',
+      tech4: 'bootstrap',
+      tech5: 'terminal',
+      tech6: 'codepen',
+    },
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+  },
+  {
+    id: 5,
+    url: 'assets/img/Snapshoot-Portfolio.png',
+    name: 'Keeping track of hundreds of components ',
+    linkVersion: '#',
+    linkSource: '#',
+    tecnologies: {
+      tech1: 'codekit',
+      tech2: 'github',
+      tech3: 'JavScript',
+      tech4: 'bootstrap',
+      tech5: 'terminal',
+      tech6: 'codepen',
+    },
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+    when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+  },
+];
+
+const cardWorkMobile = (items) => `
+  <article class="card-work-mobile">
+    <img src="assets/img/closedetailbtn.svg" alt="close detail mobile" class="close-detail-mobile">
+    <img src=${arrayCards[items].url} alt="details image" class="card-image-details-mobile">
+    <h2 class="card-name-mobile">${arrayCards[items].name}</h2>
+    <ul class="card-tags">
+      <li>${arrayCards[items].tecnologies.tech1}</li>
+      <li>${arrayCards[items].tecnologies.tech2}</li>
+      <li>${arrayCards[items].tecnologies.tech3}</li>
+    </ul>
+    <p class="card-description-mobile">${arrayCards[items].description}</p>
+    <div class="card-button-mobile">
+      <a href=${arrayCards[items].linkVersion} class="livebtn-mobile">
+        See Live
+        <img src="assets/img/liveicon.svg" alt="live icon">
+      </a>
+      <a href=${arrayCards[items].linkSource} class="sourcebtn-mobile">
+        See Source
+        <img src="assets/img/sourceicon.svg" alt="source icon">
+      </a>
+    </div>
+    <script src="index.js"></script>
+  </article>
+`;
```

Comment:
- [ ] Your popup opens and closes correctly, but on the mobile version, it is not centralized on the screen making a bad user experience. The popup opens and it is larger than the screen size, the user needs to scroll right and left, and up and down to see the content. Kindly centralize and make it responsive to every screen and display the entire content of the popup like the Figma template. (There are whitespaces between elements of the popup making it larger than it should be)
 - [ ] Also there is a gray overlay around the popup missing. Please implement the overlay so it can hide the content below the popup.
 - Your popup design:
 - 
<img width="476" alt="image" src="https://user-images.githubusercontent.com/61919617/216107921-e138f378-4937-49c7-b6b3-38693c238b1a.png">
<img width="424" alt="image" src="https://user-images.githubusercontent.com/61919617/216107977-375a3a8a-d33a-4583-9c5f-8e755317763f.png">
<img width="489" alt="image" src="https://user-images.githubusercontent.com/61919617/216108058-abc05832-b933-4417-8827-c1b109865039.png">
<img width="468" alt="image" src="https://user-images.githubusercontent.com/61919617/216108094-3e5756b3-e8e8-46f4-b247-a6da3681cdfb.png">

 - Figma template:
 - 
<img width="245" alt="image" src="https://user-images.githubusercontent.com/61919617/216108249-c75548a3-5432-47f1-902c-d28a609251ae.png">